keps/sig-api-machinery/365-paginated-lists/README.MD

```diff
@@ -275,4 +436,390 @@ For the initial alpha release, chunking would be behind a feature flag and attem
 flags should be ignored. While disabled, a `continue` token should never be returned by the server as part of a list.
 
 Future work might offer more options for clients to page in an inconsistent fashion, or allow clients to directly 
-specify the parts of the namespace / name keyspace they wish to range over (paging).
\ No newline at end of file
+specify the parts of the namespace / name keyspace they wish to range over (paging).
+
+### Test Plan
+
+- unit tests for the pagination logic in etcd3 client
+- integration tests checking valid and invalid combinations of limit and continue both
+  with watchcache enabled and disabled
+- e2e tests verifying that clients opting in for chunking can retrieve consistent
+  paginated results
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+- Initial e2e tests completed and enabled
+
+#### Beta
+
+- All tests are stable
+- Scalability impact of the feature is asessed
+
+#### GA
+
+- e2e tests are graduated to conformance
+- support for pagination is added to watchcache
```

Comment:
Do we need any docs? eg within https://k8s.dev/docs/

If we do need docs, are we happy with the quality of what we have?